examples/adafruit_io_mqtt/adafruit_io_simpletest_cellular.py

```diff
@@ -0,0 +1,120 @@
+# Example of using the Adafruit IO CircuitPython MQTT client
+# to subscribe to an Adafruit IO feed and publish random data
+# to be received by the feed.
+#
+# Example by Tony DiCola for Adafruit Industries
+# Modified by Brent Rubell for Adafruit Industries, 2019
+import time
+from random import randint
+
+import board
+import busio
+import digitalio
+
+from adafruit_fona.adafruit_fona import FONA
+from adafruit_fona.adafruit_fona_gsm import GSM
+import adafruit_fona.adafruit_fona_socket as cellular_socket
+
+from adafruit_io.adafruit_io import IO_MQTT
+import adafruit_minimqtt as MQTT
+
+# Get MQTT details and more from a secrets.py file
+try:
+    from secrets import secrets
+except ImportError:
+    print("MQTT secrets are kept in secrets.py, please add them there!")
+    raise
+
+# Create a serial connection for the FONA connection using 4800 baud.
+# These are the defaults you should use for the FONA Shield.
+# For other boards set RX = GPS module TX, and TX = GPS module RX pins.
+uart = busio.UART(board.TX, board.RX, baudrate=4800)
```

Comment:
you can try higher baud rates - 4800 was just for arduino since it was using software serial

Reply:
@ladyada  Ok - removed the baudrate parameter in favor of using the default UART API baudrate of 9600 baud (https://circuitpython.readthedocs.io/en/latest/shared-bindings/busio/UART.html#busio.UART)